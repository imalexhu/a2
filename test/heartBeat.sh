#this script tests for getting getting removed content after 12 seconds

java AggregationServer &
AGRE=$!
sleep 1
java ContentServer 0 1 50 &
CS=$!
sleep 1
java GETClient 12 &
GET=$!
sleep 10
sleep 3 
rm faultTollerance.txt
touch faultTollerance.txt
kill $AGRE
kill $GET
kill $CS