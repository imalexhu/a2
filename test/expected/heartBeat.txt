Started Aggregation Server
Started Content Server
New Content Added from CS : 0
201 - HTTP_CREATED
Started Client 

title : Life is beans and peas
link : www.beansandpeas.com
updated : 2015-08-07T18:29:02Z
id : urn:uuid:1225c695-cfb8-4ebb-aaaa-80da344efa6b
summary : Sometimes I wonder if there are more to life than just beans and peas

Content Expired
GET Attempt 0 failed after 0 seconds
